suite-native/receive/src/components/ConfirmOnTrezorImage.tsx

```diff
@@ -13,25 +14,30 @@ const imageContainerStyle = prepareNativeStyle(utils => ({
 }));
 
 export const ConfirmOnTrezorImage = () => {
+    const [isBottomSheetOpened, setIsBottomSheetOpened] = useState(false);
     const { applyStyle } = useNativeStyles();
 
-    const { showToast } = useToast();
+    const handleImagePress = () => {
+        setIsBottomSheetOpened(true);
+    };
 
-    // TODO: https://github.com/trezor/trezor-suite/issues/9776
-    const handlePress = () => {
-        showToast({
-            variant: 'default',
-            message: 'TODO: show informational modal',
-            icon: 'warningCircle',
-        });
+    const handleCloseBottomSheet = () => {
+        setIsBottomSheetOpened(false);
     };
 
     return (
-        // TODO: images will be revisited in issue: https://github.com/trezor/trezor-suite/issues/9777
-        <Animated.View entering={FadeIn} style={applyStyle(imageContainerStyle)}>
-            <Pressable onPress={handlePress}>
-                <Image source={require('../../assets/confirmOnTrezor.png')} />
-            </Pressable>
-        </Animated.View>
+        <>
+            {/* TODO: images will be revisited in issue:
+            https://github.com/trezor/trezor-suite/issues/9777 */}
+            <Animated.View entering={FadeIn} style={applyStyle(imageContainerStyle)}>
+                <Pressable onPress={handleImagePress}>
+                    <Image source={require('../../assets/confirmOnTrezor.png')} />
```

Comment:
Can you put this to useMemo?

Reply:
fixed: https://github.com/trezor/trezor-suite/pull/9934/commits/9eec510f27b9b4bcf4b7113aa1019dfca9c68541